tests/common.go

```diff
@@ -4338,18 +4349,23 @@ func CreateCloudCredential(provider, credName string, uid, orgID string, ctx con
 }
 
 // CreateS3BackupLocation creates backup location for S3
-func CreateS3BackupLocation(name string, uid, cloudCred string, cloudCredUID string, bucketName string, orgID string, encryptionKey string) error {
+func CreateS3BackupLocation(name, uid, cloudCred, cloudCredUID, bucketName, orgID, encryptionKey string, validate bool, sseS3EncryptionType ...api.S3Config_Sse) error {
```

Comment:
The optional parameter needs to be added to the function `CreateS3BackupLocationWithContext`